### PR TITLE
Update crates and dsc version

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
# PR Summary

Update version to preview.11
Update crates to latest versions
Fix code due to crate changes

### CODE REVIEW

File: dsc/src/main.rs [hunk 1]
```diff
@@ -187,7 +187,7 @@ Press any key to close this window
     };
 
     // MS Store runs app using `sihost.exe`
-    if parent_process.name().to_lowercase() == "sihost.exe" || parent_process.name().to_lowercase() == "explorer.exe"{
+    if parent_process.name().to_ascii_lowercase() == "sihost.exe" || parent_process.name().to_ascii_lowercase() == "explorer.exe"{
```
Comment (anmenaga) [inline dsc/src/main.rs:190]: What is the reason for this change?

**Review by anmenaga — APPROVED** [review]

File: dsc/src/main.rs (hunk 1 above)
Comment (SteveL-MSFT) [reply dsc/src/main.rs:190]: The crate version changed and the old method was renamed
